### PR TITLE
reduce spnego FAT trace

### PR DESCRIPTION
**Pull request by Zech-Hein**
Reducing trace to improve FAT performance, should stop Build Break Defect 282469 from occurring.

The full FAT on spnego was generating over 6GB of trace.

Before it was using the following spec on most test servers:
```
com.ibm.ws.logging.trace.specification=*=info=enabled:\
com.ibm.ws.security.*=all=enabled:\
com.ibm.ws.webcontainer.security.*=all=enabled:\
```
This PR reduces it by using the following instead:
```
com.ibm.ws.logging.trace.specification=*=info=enabled:\
com.ibm.ws.security.spnego*=all=enabled:\
```

From my local testing, this should reduce duration of most test from ~45 to ~35 seconds

and reduce the huge tracing output by ~99.8%
from ~117MB to 150KB per test server

### CODE REVIEW

**Comment by Zech-Hein** [comment]
#build

**Review by utle — APPROVED** [review]

**Comment by Zech-Hein** [comment]
PB had an infrastructure build problem and did not run the FAT. re-running to hopefully get a clean build.
#build

**Comment by Zech-Hein** [comment]
#build

**Comment by Zech-Hein** [comment]
```
Exception was thrown: Installed feature(s) [appsecurity-4.0, cdi-3.0] were not defined in the autoFVT/fat-metadata.json file! To correct this, add [appsecurity-4.0, cdi-3.0] to the 'tested.features' property in the bnd.bnd or build-test.xml file for this FAT so that an accurate test depdendency graph can be generated in the future.
	at com.ibm.ws.security.spnego.fat.ServicePrincipalNamesTest.testSingleSpn(ServicePrincipalNamesTest.java:83)
```

@jvanhill  is adding the features to the `tested.features:` field to fix the failure, will rebuild this PR when his change is in

**Comment by Zech-Hein** [comment]
tested.features exception fixed in https://github.com/OpenLiberty/open-liberty/pull/16701

**Comment by Zech-Hein** [comment]
#libby

**Comment by Zech-Hein** [comment]
#build

File: dev/com.ibm.ws.security.spnego_fat.1/publish/servers/BackendServer/bootstrap.properties [hunk 1]
```diff
@@ -12,8 +12,7 @@
 bootstrap.include=../testports.properties
 
 com.ibm.ws.logging.trace.specification=*=info=enabled:\
-com.ibm.ws.security.*=all=enabled:\
-com.ibm.ws.webcontainer.security.*=all=enabled:\ 
+com.ibm.ws.security.spnego*=all=enabled:\
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.spnego_fat.1/publish/servers/BackendServer/bootstrap.properties]:
```suggestion
com.ibm.ws.security.spnego*=all=enabled
```

File: dev/com.ibm.ws.security.spnego_fat.1/publish/servers/DynamicSpnegoConfigTest/bootstrap.properties [hunk 2]
```diff
@@ -12,8 +12,7 @@
 bootstrap.include=../testports.properties
 
 com.ibm.ws.logging.trace.specification=*=info=enabled:\
-com.ibm.ws.security.*=all=enabled:\
-com.ibm.ws.webcontainer.security.*=all=enabled:\ 
+com.ibm.ws.security.spnego*=all=enabled:\
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.spnego_fat.1/publish/servers/DynamicSpnegoConfigTest/bootstrap.properties]:
```suggestion
com.ibm.ws.security.spnego*=all=enabled
```

File: dev/com.ibm.ws.security.spnego_fat.1/publish/servers/com.ibm.ws.security.spnego.fat.setup/bootstrap.properties [hunk 3]
```diff
@@ -12,8 +12,7 @@
 bootstrap.include=../testports.properties
 
 com.ibm.ws.logging.trace.specification=*=info=enabled:\
-com.ibm.ws.security.*=all=enabled:\
-com.ibm.ws.webcontainer.security.*=all=enabled:\ 
+com.ibm.ws.security.spnego*=all=enabled:\
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.spnego_fat.1/publish/servers/com.ibm.ws.security.spnego.fat.setup/bootstrap.properties]:
```suggestion
com.ibm.ws.security.spnego*=all=enabled
```

File: dev/com.ibm.ws.security.spnego_fat.2/publish/servers/AuthFilterElementTest/bootstrap.properties [hunk 4]
```diff
@@ -12,8 +12,7 @@
 bootstrap.include=../testports.properties
 
 com.ibm.ws.logging.trace.specification=*=info=enabled:\
-com.ibm.ws.security.*=all=enabled:\
-com.ibm.ws.webcontainer.security.*=all=enabled:\ 
+com.ibm.ws.security.spnego*=all=enabled:\
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.spnego_fat.2/publish/servers/AuthFilterElementTest/bootstrap.properties]:
```suggestion
com.ibm.ws.security.spnego*=all=enabled
```

File: dev/com.ibm.ws.security.spnego_fat.2/publish/servers/DynamicAuthFilterTest/bootstrap.properties [hunk 5]
```diff
@@ -12,8 +12,7 @@
 bootstrap.include=../testports.properties
 
 com.ibm.ws.logging.trace.specification=*=info=enabled:\
-com.ibm.ws.security.*=all=enabled:\
-com.ibm.ws.webcontainer.security.*=all=enabled:\ 
+com.ibm.ws.security.spnego*=all=enabled:\
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.spnego_fat.2/publish/servers/DynamicAuthFilterTest/bootstrap.properties]:
```suggestion
com.ibm.ws.security.spnego*=all=enabled
```

File: dev/com.ibm.ws.security.spnego_fat.2/publish/servers/InvokeAfterSSOTest/bootstrap.properties [hunk 6]
```diff
@@ -12,8 +12,7 @@
 bootstrap.include=../testports.properties
 
 com.ibm.ws.logging.trace.specification=*=info=enabled:\
-com.ibm.ws.security.*=all=enabled:\
-com.ibm.ws.webcontainer.security.*=all=enabled:\ 
+com.ibm.ws.security.spnego*=all=enabled:\
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.spnego_fat.2/publish/servers/InvokeAfterSSOTest/bootstrap.properties]:
```suggestion
com.ibm.ws.security.spnego*=all=enabled
```

File: dev/com.ibm.ws.security.spnego_fat.2/publish/servers/com.ibm.ws.security.spnego.fat.setup/bootstrap.properties [hunk 7]
```diff
@@ -12,8 +12,7 @@
 bootstrap.include=../testports.properties
 
 com.ibm.ws.logging.trace.specification=*=info=enabled:\
-com.ibm.ws.security.*=all=enabled:\
-com.ibm.ws.webcontainer.security.*=all=enabled:\ 
+com.ibm.ws.security.spnego*=all=enabled:\
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.spnego_fat.2/publish/servers/com.ibm.ws.security.spnego.fat.setup/bootstrap.properties]:
```suggestion
com.ibm.ws.security.spnego*=all=enabled
```

File: dev/com.ibm.ws.security.spnego_fat/publish/servers/BasicAuthTest/bootstrap.properties [hunk 8]
```diff
@@ -12,8 +12,7 @@
 bootstrap.include=../testports.properties
 
 com.ibm.ws.logging.trace.specification=*=info=enabled:\
-com.ibm.ws.security.*=all=enabled:\
-com.ibm.ws.webcontainer.security.*=all=enabled:\ 
+com.ibm.ws.security.spnego*=all=enabled:\
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.spnego_fat/publish/servers/BasicAuthTest/bootstrap.properties]:
```suggestion
com.ibm.ws.security.spnego*=all=enabled
```

File: dev/com.ibm.ws.security.spnego_fat/publish/servers/CanonicalHostNameTest/bootstrap.properties [hunk 9]
```diff
@@ -12,8 +12,7 @@
 bootstrap.include=../testports.properties
 
 com.ibm.ws.logging.trace.specification=*=info=enabled:\
-com.ibm.ws.security.*=all=enabled:\
-com.ibm.ws.webcontainer.security.*=all=enabled:\ 
+com.ibm.ws.security.spnego*=all=enabled:\
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.spnego_fat/publish/servers/CanonicalHostNameTest/bootstrap.properties]:
```suggestion
com.ibm.ws.security.spnego*=all=enabled
```

File: dev/com.ibm.ws.security.spnego_fat/publish/servers/IncludeClientGSSCredentialInSubjectTest/bootstrap.properties [hunk 10]
```diff
@@ -12,8 +12,7 @@
 bootstrap.include=../testports.properties
 
 com.ibm.ws.logging.trace.specification=*=info=enabled:\
-com.ibm.ws.security.*=all=enabled:\
-com.ibm.ws.webcontainer.security.*=all=enabled:\ 
+com.ibm.ws.security.spnego*=all=enabled:\
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.spnego_fat/publish/servers/IncludeClientGSSCredentialInSubjectTest/bootstrap.properties]:
```suggestion
com.ibm.ws.security.spnego*=all=enabled
```

File: dev/com.ibm.ws.security.spnego_fat/publish/servers/IncludeCustomCacheKeyTest/bootstrap.properties [hunk 11]
```diff
@@ -12,8 +12,7 @@
 bootstrap.include=../testports.properties
 
 com.ibm.ws.logging.trace.specification=*=info=enabled:\
-com.ibm.ws.security.*=all=enabled:\
-com.ibm.ws.webcontainer.security.*=all=enabled:\ 
+com.ibm.ws.security.spnego*=all=enabled:\
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.spnego_fat/publish/servers/IncludeCustomCacheKeyTest/bootstrap.properties]:
```suggestion
com.ibm.ws.security.spnego*=all=enabled
```

File: dev/com.ibm.ws.security.spnego_fat/publish/servers/Krb5ConfigTest/bootstrap.properties [hunk 12]
```diff
@@ -12,8 +12,7 @@
 bootstrap.include=../testports.properties
 
 com.ibm.ws.logging.trace.specification=*=info=enabled:\
-com.ibm.ws.security.*=all=enabled:\
-com.ibm.ws.webcontainer.security.*=all=enabled:\ 
+com.ibm.ws.security.spnego*=all=enabled:\
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.spnego_fat/publish/servers/Krb5ConfigTest/bootstrap.properties]:
```suggestion
com.ibm.ws.security.spnego*=all=enabled
```

File: dev/com.ibm.ws.security.spnego_fat/publish/servers/ServicePrincipalNamesTest/bootstrap.properties [hunk 13]
```diff
@@ -12,8 +12,7 @@
 bootstrap.include=../testports.properties
 
 com.ibm.ws.logging.trace.specification=*=info=enabled:\
-com.ibm.ws.security.*=all=enabled:\
-com.ibm.ws.webcontainer.security.*=all=enabled:\ 
+com.ibm.ws.security.spnego*=all=enabled:\
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.spnego_fat/publish/servers/ServicePrincipalNamesTest/bootstrap.properties]:
```suggestion
com.ibm.ws.security.spnego*=all=enabled
```

File: dev/com.ibm.ws.security.spnego_fat/publish/servers/SpnegoTokenHelperTest/bootstrap.properties [hunk 14]
```diff
@@ -12,9 +12,8 @@
 bootstrap.include=../testports.properties
 
 com.ibm.ws.logging.trace.specification=*=info=enabled:\
-com.ibm.ws.security.*=all=enabled:\
+com.ibm.ws.security.spnego*=all=enabled:\
 com.ibm.wsspi.security.*=all=enabled:\
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.spnego_fat/publish/servers/SpnegoTokenHelperTest/bootstrap.properties]:
```suggestion
com.ibm.wsspi.security.*=all=enabled
```

File: dev/com.ibm.ws.security.spnego_fat/publish/servers/com.ibm.ws.security.spnego.fat.formlogin/bootstrap.properties [hunk 15]
```diff
@@ -12,8 +12,7 @@
 bootstrap.include=../testports.properties
 
 com.ibm.ws.logging.trace.specification=*=info=enabled:\
-com.ibm.ws.security.*=all=enabled:\
-com.ibm.ws.webcontainer.security.*=all=enabled:\ 
+com.ibm.ws.security.spnego*=all=enabled:\
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.spnego_fat/publish/servers/com.ibm.ws.security.spnego.fat.formlogin/bootstrap.properties]:
```suggestion
com.ibm.ws.security.spnego*=all=enabled
```

File: dev/com.ibm.ws.security.spnego_fat/publish/servers/com.ibm.ws.security.spnego.fat.setup/bootstrap.properties [hunk 16]
```diff
@@ -12,8 +12,7 @@
 bootstrap.include=../testports.properties
 
 com.ibm.ws.logging.trace.specification=*=info=enabled:\
-com.ibm.ws.security.*=all=enabled:\
-com.ibm.ws.webcontainer.security.*=all=enabled:\ 
+com.ibm.ws.security.spnego*=all=enabled:\
```
Comment (jvanhill) [inline dev/com.ibm.ws.security.spnego_fat/publish/servers/com.ibm.ws.security.spnego.fat.setup/bootstrap.properties]:
```suggestion
com.ibm.ws.security.spnego*=all=enabled
```

**Review by jvanhill — APPROVED** [review]
I am approving. I would prefer that the trace specs not have continuation lines after the last specification as if another line is added it may not be intended to be a continuation. I have marked all the instances I saw.

**Comment by Zech-Hein** [comment]
#libby

**Comment by Zech-Hein** [comment]
resubmitting build with last commit 
#build

**Pull request merged**